modules/gathering-data-specific-features.adoc

```diff
@@ -57,6 +57,12 @@ endif::openshift-dedicated[]
 |`registry.redhat.io/openshift-sandboxed-containers/osc-must-gather-rhel8:1.2.0`
 |Data collection for {sandboxed-containers-first}.
 
+|`registry.redhat.io/openshift4/poison-pill-must-gather-rhel8:v0.3.0`
+|Data collection for the Poison Pill Operator and the Node Health Check Operator.
+
+|`registry.redhat.io/workload-availability/node-maintenance-must-gather-rhel8:v4.10.0`
```

Comment:
interesting... we do NOT have a `v4.10` for this one 🤔

Reply:
@anna-savina, @slintes and @prabinovRedhat 
Updated the image name per our discussion. PTAL and ack. Thank you!